readme.md

```diff
@@ -77,3 +76,41 @@ The docker containers and images created by the tests framework are removed when
 The docker images and containers have to be removed manually if CLEAN_UP_TEST_ENVIRONMENT=false. The Pyrsia integration
 tests also create the temp directory `/tmp/pyrsia_tests`which is not removed by the tests framework and if necessary has to be removed
 manually.
+
+## Troubleshooting
+
+In case of any problems with the tests environment reset the enviroment as follow:
+1) Remove all docker images and containers:
+   ```sh
+      docker system prune --all
+   ```
+2) Remove the integration tests temp directory:
+   ```sh
+      rm -rf /tmp/pyrsia_tests/
+   ```
+
+## Tests logger
+
+Supported logging levels:
+- INFO (default)
+- DEBUG
+- ERROR
+
+How to use logger in the tests:
+
+```sh
+  # load the test from the library
+  load '../lib/logger/load'
```

Comment:
This is already in `common-setup.bash`, so it is not needed anymore?

Reply:
I just want to mention how to add the logger in case that it's used somehow outside of the tests (no common setup) or particular test doesn't use the common setup which is also possible. In general if this library is added more than once it's OK and IMHO the docs should mention how to use it/load it.